src/components/Catalog.vue

```diff
@@ -432,7 +433,11 @@ export default {
         key = "title";
       }
 
-      if (typeof a[key] === "number" && typeof b[key] === "number") {
+      if (typeof a[key] === "undefined" || a[key] === null) {
+        return -1;
+      } else if (typeof b[key] === "undefined" || b[key] === null) {
```

Comment:
These should be able to be simplified as `_[key] == null` (double `=` equates `null` and `undefined`).

Reply:
I usually like to be explicit, but indeed it's possible to simplify. Updated the code and squashed the commits.